Content.Shared/VendingMachines/SharedVendingMachineComponent.cs

```diff
@@ -97,4 +97,6 @@ public enum EjectWireKey : byte
     }
 
     public sealed class VendingMachineSelfDispenseEvent : InstantActionEvent { };
+
+    public sealed class VendingMachineRestockEvent : HandledEntityEventArgs { };
```

Comment:
Comment?

I would've assumed this is raised when it's successful but then I saw it inherits from handled.